src/utils/index.ts

```diff
@@ -1,2 +1,3 @@
-// eslint-disable-next-line import/prefer-default-export
 export { default as getHoverColor } from './get-hover-color';
+export { default as isTransparentColor } from './is-transparent-color';
+export { default as isDarkColor } from './is-dark-color';
```

Comment:
Sure, if they are related, definitely we can put them in one file

Reply:
Would you prefer to export them via the util index as separate function or part of a `colorUtils` object?

Reply:
```js
export { default as getHoverColor } from './color-utils';
export { default as isTransparentColor } from './color-utils';
export { default as isDarkColor } from './color-utils';
export { default as COLORING } from './color-utils';
export { default as focusSelectionToolbar } from './focus-selection-toolbar';
export { default as preventDefaultBehavior } from './prevent-default-behavior';
export { default as handleLastTab } from './handle-last-tab';
```

OR

```js
export { default as colorUtils } from './color-utils';
export { default as focusSelectionToolbar } from './focus-selection-toolbar';
export { default as preventDefaultBehavior } from './prevent-default-behavior';
export { default as handleLastTab } from './handle-last-tab';
```

Reply:
`colorUtils` object seems clean and only one object that is responsible for all. so second one is looks good 👍

Reply:
That might just make it harder to tree-shake. But I'll see what I can do

Reply:
The import would be `import { colorUtils } from "@qlik/nebula-table-utils/lib/utils";` and then use it `colorUtils.isDarkColor("red")`?

Reply:
Probably tree shaking is not a big issue with rollup since it takes care of it as it doc says at here: https://rollupjs.org/introduction/#tree-shaking

I agree that this: 
```
colorUtils.isDarkColor("red")
```
is kinda ichy but we destructure things where we use them